src/purge.rs

```diff
@@ -194,6 +238,35 @@ where
             }
         }
     }
+
+    fn squeeze_rewrite_queue(&self) {
+        let memtables = self
+            .memtables
+            .collect(|t| t.min_file_num(LogQueue::Rewrite).unwrap_or_default() > 0);
```

Comment:
It means that all of data in `Rewrite` queue will be rewritten again? I do not think it is a  good idea.... At least most of them may be  in current write file and they shall not be rewritten. How about only rewriting the oldest file? Because we only allowed 10% of rewrite queue keep, the wal file may not exceed 2(both of them are 128MB size). So let we just rewrite the oldest file and keep the current file.